system/Autoloader/Autoloader.php

```diff
@@ -380,6 +383,7 @@ private function loadComposerNamespaces(ClassLoader $composer, array $composerPa
             }
         }
 
+        // @phpstan-ignore-next-line
```

Comment:
pls use `@phpstan-ignore` here with the error type you are suppressing

Reply:
Ok